site/docs/guides/developer/index.md

```diff
@@ -224,6 +225,42 @@ The modules for each registered service that need to run in the OMAG Server Plat
 
 The best guide for building registered services are the existing implementations found in [egeria.git](https://github.com/odpi/egeria/tree/main/open-metadata-implementation){ target=gh }.  You can see the way the code is organized and the services that they depend on.
 
+## Build considerations
+
+Egeria's best practise is to use [Gradle](https://gradle.org/) to build the Egeria extensions.
+
+Principles around constructing the Gradle build file:
+
+- build against mavenCentral and snapshots if required. 
+
+```
+ repositories {
+    mavenCentral()
+    maven { url("https://oss.sonatype.org/content/repositories/snapshots") }
+    // uncomment to pick up from local ~/.m2 - but can be unpredictable. recommend to use maven central and snapshots
+    //    mavenLocal()
+  }
+```
+
+It is possible to pick up content from local Maven by uncommenting in the above snippet. This may be required during development if the dependant change is not in a snapshot or release.
+For example your new Egeria connector requires changes to an OMAS that you are also developing, but have not yet been merged.
+
+- versions should be defined as variables
+- the Egeria version should up to date when the extension is originally contributed. This version and other associated technology versions can be
```

Comment:
grammar - `Egeria version should up to date` -> `Egeria version should be up-to-date`